moviedb_project/moviedb_project/settings.py

```diff
@@ -37,6 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'crispy_forms',
```

Comment:
This is just a bootstrap django form I found, you can implement the same thing manually directly on the template if you wanted something more custom but this is a quick/dirty way to get a prettified form.